packages/react-components/src/components/styles/InputBoxComponent.style.ts

```diff
@@ -78,14 +78,12 @@ export const textFieldStyle = (errorColor: string, hasErrorMessage: boolean, dis
  */
 export const inputButtonStyle = mergeStyles({
   color: 'grey',
-  cursor: 'pointer',
```

Comment:
nit: I would have preferred these style changes to be in a separate PR.

Most of the e2e snapshot changes are due to these, unrelated to the non-trivial code changes elsewhere in this PR.

Reply:
This is part of the PR. Styles need to be changed due to using IconButton